tests/SAHB.GraphQLClient.Tests/Batching/BatchingTest.cs

```diff
@@ -266,5 +265,54 @@ public async Task Test_Execute_Two_Times_Should_Not_Throw_Exception()
             Assert.Equal(result1.Part1Field1, result2.Part1Field1);
             Assert.Equal(result1.Part1Field2, result2.Part1Field2);
         }
+
+        [Fact]
+        public async Task Test_ExecuteDetailed_Returns_Expected_Headers_And_Data()
```

Comment:
Please also add a testcase which uses the GraphQLHttpClient class. This could be placed in the SAHB.GraphQLClient.Tests.GraphQLClient.GraphQLClientTest class as a new testcase.